java_fibonacci.java

```diff
@@ -0,0 +1,25 @@
+import java.util.Scanner;
+
+public class java_fibonacci{
```

Comment:
Codacy found an issue: [The class name 'java_fibonacci' doesn't match '[A-Z][a-zA-Z0-9]*'](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6353698)